src/mcpil.py

```diff
@@ -538,7 +583,7 @@ def main():
     window.resizable(True, True)
 
     # Set icon in taskbar
-    window.iconphoto(True, PhotoImage(file="/usr/share/pixmap/mcpil.png"))
+    window.iconphoto(True, PhotoImage(file="/usr/share/pixmaps/mcpil.png"))
```

Comment:
What does this change?

Reply:
Ooh, this makes the image actually appear without errors, rpinews or alv mistyped

Reply:
This is correct, probably RPiNews or me made a typo.

Reply:
Lol